<launch>

  <!-- rviz -->
  <!-- <node pkg="rviz" type="rviz" name="rviz" args="-d $(find handyman)/launch/handyman.rviz"/> -->
  
  <arg name="sub_msg_to_robot_topic_name"       default="/handyman/message/to_robot" />
  <arg name="pub_msg_to_moderator_topic_name"   default="/handyman/message/to_moderator" />
  <arg name="sub_joint_state_topic_name"        default="/hsrb/joint_states" />
  <arg name="pub_base_twist_topic_name"         default="/hsrb/command_velocity" />
  <arg name="pub_arm_trajectory_topic_name"     default="/hsrb/arm_trajectory_controller/command" />
  <arg name="pub_gripper_trajectory_topic_name" default="/hsrb/gripper_controller/command" />
	
  <node name="start_handy" pkg="tamhome_handyman" type="main.py"  >
    <param name="sub_msg_to_robot_topic_name"       type="string" value="$(arg sub_msg_to_robot_topic_name)" />
    <param name="pub_msg_to_moderator_topic_name"   type="string" value="$(arg pub_msg_to_moderator_topic_name)" />
    <param name="pub_base_twist_topic_name"         type="string" value="$(arg pub_base_twist_topic_name)" />
    <param name="pub_arm_trajectory_topic_name"     type="string" value="$(arg pub_arm_trajectory_topic_name)" />
    <param name="pub_gripper_trajectory_topic_name" type="string" value="$(arg pub_gripper_trajectory_topic_name)" />
  </node>

</launch>
